src/map/options.ts

```diff
@@ -100,9 +100,9 @@ export class MapOptions extends OptionsGroup {
         this.size = {
             factor : new HTMLOption('number', 50),
             mode : new HTMLOption('string', 'constant'),
-            property : new HTMLOption('string', ''),
+            property : new HTMLOption('string', propertiesName[0]),
         };
-        this.size.property.validate = optionValidator(propertiesName.concat(['']), 'size');
+        this.size.property.validate = optionValidator(propertiesName, 'size');
```

Comment:
This is breaking the Arginine-Dipeptide example, as well as any files saved with `size: {property: ''}` in the settings (in particular the KPCovR ones), which is a bit unfortunate 😐 

The best fix I see is to catch the errors when applying settings & convert them to warnings, but it would be a bit noisy. Alternatively, we could manually deal with this case for backward compatibility.

Reply:
Fixed in 99ca9c9